Update PosteItaliane

frmayall1978 opened the pull request:
Added a 2fa options supported by PosteItaliane; they do allow to 2fa yourself via SMS and their stupid app.

Commit:
Update poste.it.json

Added a 2fa options supported by PosteItaliane; they do allow to 2fa yourself via SMS and their stupid app.

Signed-off-by: frmayall1978 <90851285+frmayall1978@users.noreply.github.com>

Commit:
Update poste.it.json

Signed-off-by: frmayall1978 <90851285+frmayall1978@users.noreply.github.com>

fletcherdodd commented on entries/p/poste.it.json:
```diff
@@ -1,13 +1,14 @@
+    "custom-software": [
+      "PosteItaliane App"
+    ],
     "url": "https://www.poste.it/",
```
```suggestion
```
No need for the URL as it is the same as domain field.

Commit:
Update entries/p/poste.it.json

Removed URL field, as it's the same as domain name.

Co-authored-by: Fletcher Dodd <fletcherdodd@outlook.com>
Signed-off-by: frmayall1978 <90851285+frmayall1978@users.noreply.github.com>

Commit:
Update PosteItaliane

Added documentation on 2FA for the service. There is no English version, so you may have to trust me on this.

Signed-off-by: frmayall1978 <90851285+frmayall1978@users.noreply.github.com>

Commit:
Update PosteItaliane

Corrected JSON syntax error. It's been a tough day.

Signed-off-by: frmayall1978 <90851285+frmayall1978@users.noreply.github.com>
